gregory-2019-cell.tei.xml

```diff
@@ -0,0 +1,5292 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TEI xml:space="preserve" xmlns="http://www.tei-c.org/ns/1.0"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://www.tei-c.org/ns/1.0 /home/lopez/grobid/grobid-home/schemas/xsd/Grobid.xsd"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+	<teiHeader xml:lang="en">
+		<fileDesc>
+			<titleStmt>
+				<title level="a" type="main">17 Cé line Dimier, 6,10,17 Sté phane Pesant, 14,15 Marc Picheral, 6,17 Sergey Pisarev, 16 Julie Poulain, 5,17 Jean-É ric Tremblay, 11 Dean Vik, 1 Tara Oceans Coordinators, Marcel Babin, 11 Chris Bowler</title>
```

Comment:
Title is incorrect. Curious how certain the model is that this is the title.

![image](https://user-images.githubusercontent.com/1177031/83211974-1ef71980-a0fa-11ea-8032-b67b4064e5ed.png)